### PR TITLE
fixed issue #515 and added test case for same

### PR DESCRIPTION
**Pull request by AnishShah**

### CODE REVIEW

File: bookie/tests/test_bcelery/test_bcelery.py [hunk 1]
```diff
@@ -7,6 +9,8 @@
 from bookie.models import Tag
 from bookie.models import stats
 from bookie.models.auth import User
+from bookie.models.auth import UserMgr
```
Comment (mitechie) [inline bookie/tests/test_bcelery/test_bcelery.py]: Please do the imports together like

```
 from bookie.models.auth import (
     Activation,
     UserMgr,
)
```

File: bookie/tests/test_bcelery/test_bcelery.py [hunk 2]
```diff
@@ -110,3 +114,34 @@ def test_process_twitter_connections(self, mock_create_twitter_api):
...
         self.assertTrue(mock_create_twitter_api.called)
+
+    def test_task_delete_non_activated_account(self):
+        """The task should delete non activated users"""
+        email = u'testingdelete@gmail.com'
+        new_user = UserMgr.signup_user(email, u'testcase')
+        users = User.query.all()
+        activations = Activation.query.all()
+        self.assertEqual(
+            4,
```
Comment (mitechie) [inline bookie/tests/test_bcelery/test_bcelery.py:127]: you've got 4 here, but the message is 3 users?

File: bookie/tests/test_bcelery/test_bcelery.py [hunk 3]
```diff
@@ -110,3 +114,34 @@ def test_process_twitter_connections(self, mock_create_twitter_api):
...
+            len(users),
+            'We should have a total of 3 users : ' + str(len(users))
+        )
+        self.assertEqual(
+            3,
```
Comment (mitechie) [inline bookie/tests/test_bcelery/test_bcelery.py:132]: same here, 3 activations but a message of 2 expected. Can you list out the expected state here?

**Comment by mitechie** [comment]
This looks ok but the test needs some clarification.

File: bookie/tests/test_bcelery/test_bcelery.py (hunk 2 above)
Comment (AnishShah) [inline bookie/tests/test_bcelery/test_bcelery.py:127]: @mitechie very sorry. The message should be 4 users. Sorry for the typo.

**Comment by AnishShah** [comment]
@mitechie  Sorry for the typos. I have fixed the import thing too.

**Comment by mitechie** [comment]
Tests pass but lint fails:

```
bookie/tests/test_bcelery/test_bcelery.py:136:80: E501 line too long (80 > 79 characters)
```

After that this can be merged. Thanks for the great work.

**Comment by AnishShah** [comment]
Damn. I have fixed it.

**Comment by mitechie** [comment]
Thanks for the update. If you have not already, please make sure to add yourself to the credits file!

**Pull request merged**